concepts/iteration.coral

```diff
@@ -0,0 +1,18 @@
+/*
+Contains example iterations.
+*/
+
+// For loop
+for (let i = 0; condition; i++) {
+    ...
+}
+
+// Foreach loop
+foreach item in items {
```

Comment:
All of the other iteration loops have parentheses around them. I think for consistency we should do that here too:
```suggestion
foreach (item in items) {
```